modin/experimental/engines/pandas_on_ray/sql.py

```diff
@@ -184,6 +184,7 @@ def get_query_info(sql, con, partition_column):
         cols = get_query_columns(engine, query)
     # TODO allow validation that takes into account edge cases of pandas e.g. "[index]"
     # check_partition_column(partition_column, cols)
+    # partition_column isn't used
```

Comment:
This should be a `FIXME` or smth comment (with a call-to-action) describing what to do